alf/algorithms/mcts_algorithm.py

```diff
@@ -279,6 +313,11 @@ def __init__(
                 using reverse KL divergence will be used for tree search.
             learn_with_exploration_policy (bool): If True, a policy calculated
                 using reverse KL divergence will be used for learning.
+            max_unroll_length (int): maximal allowed unroll steps when building
+                the search tree.
```

Comment:
Would be better if we state the connection between `max_unroll_length` and the maximum tree depth. This would help clarify to the reader how it affects the tree search.